src/rebloom.c

```diff
@@ -960,37 +960,37 @@ int RedisModule_OnLoad(RedisModuleCtx *ctx, RedisModuleString **argv, int argc)
     }
 
     if (argc % 2) {
-        BAIL("Invalid number of arguments passed");
+        BAIL("Invalid number of arguments passed", NULL); // LCOV_EXCL_LINE
     }
 
     for (int ii = 0; ii < argc; ii += 2) {
         if (!rsStrcasecmp(argv[ii], "initial_size")) {
             long long v;
             if (RedisModule_StringToLongLong(argv[ii + 1], &v) == REDISMODULE_ERR) {
-                BAIL("Invalid argument for 'INITIAL_SIZE'");
+                BAIL("Invalid argument for 'INITIAL_SIZE'", NULL); // LCOV_EXCL_LINE
```

Comment:
@ashtul why did you mark this line as "LCOV_EXCL_LINE"?

Reply:
@gkorland 
Since there is no way to test this without crashing the test.
This makes it clear it isn't tested and isn't meant to be tested.